scripts/proguard.pro

```diff
@@ -17,6 +17,8 @@
 
 -keep class baritone.api.** { *; } # this is the keep api
 
+-keep class baritone.Baritone
+
```

Comment:
Why?

Reply:
so i can do
BaritoneAPI.getProvider().getAllBaritones().add(new Baritone(my_custom_player_context))
in my custom utility mod and not have the Baritone class get renamed by proguard

Reply:
Just curious, what is the use case for a different player context this way?

Reply:
allows you to control multiple players from one minecraft instance since it removes the limit of only having a primary player context